collectors/tc.plugin/tc-qos-helper.sh.in

```diff
@@ -240,7 +240,7 @@ show_tc() {
 		show_fireqos_names "${x}" || show_tc_cls "${x}"
```

Comment:
There are `echo`s inside the show_fireqos_names that are not guarded with `|| exit`. They can fail in the same way if the pipe is closed.

Reply:
We did not have any error reported with them, but I agree this can happen. I had doubt about the real necessity to change these lines. I will change they now and commit in few minutes.

Reply:
Done!